addons/l10n_ae/models/account_chart_template.py

```diff
@@ -37,3 +37,31 @@ def _load_template(self, company, code_digits=None, account_ref=None, taxes_ref=
                 {'property_tax_payable_account_id': account_ref.get(self.env.ref('l10n_ae.uae_account_202003').id),
                  'property_tax_receivable_account_id': account_ref.get(self.env.ref('l10n_ae.uae_account_100103').id)})
         return account_ref, taxes_ref
+
+    @api.model
+    def _create_cash_discount_loss_account(self, company, code_digits):
+        if not self == self.env.ref('l10n_ae.uae_chart_template_standard'):
+            return super()._create_cash_discount_loss_account(company, code_digits)
+        cash_discount_loss_account = self.env['account.account'].search([('company_id', '=', company.id), ('code', 'like', '400071')], limit=1)
```

Comment:
This is a bad implementation regarding the chart template logic. It should be fields on the chart template instead